cubi_tk/snappy/pull_data_common.py

```diff
@@ -175,7 +175,7 @@ def sort_irods_object_by_date_in_path(self, irods_obj_list):
         /sodarZone/projects/../<PROJECT_UUID>/.../assay_<ASSAY_UUID>/<LIBRARY_NAME>/.../<DATE>/...
 
         :param irods_obj_list: List of iRODS objects derived from collection in SODAR.
-        :type irods_obj_list: List[IrodsDataObject]
+        :type irods_obj_list: List[DataObject]
```

Comment:
I think the correct calls/type name here is `iRODSDataObject` (at least its imported under that name in check.py; inside irods it iirc called `data_object`)